### ISSUE TITLE
Game crashes on start (1.19.2)

### ISSUE THREAD

**Issue by srnyx** [emilcarr/fabric-sneakhold#11]
https://paste.srnyx.xyz/epirefawep.properties

**Comment by emilcarr** [comment]
Looks like the minecraft codebase has changed enough that my mixin is no longer matching the right function.

I've not touched this project since I released it, please feel free to fork it, the code is relatively simple.

**Comment by srnyx** [comment]
> Looks like the minecraft codebase has changed enough that my mixin is no longer matching the right function.
> 
> I've not touched this project since I released it, please feel free to fork it, the code is relatively simple.

Yeah it's all good now, someone else forked it for me, thank you!
